CONFIGURATION.md

```diff
@@ -0,0 +1,84 @@
+# Configuration examples
+On this page we will collect some configuration examples. This is meant to help you configure debug templates for your editor. The examples are provided in JSON format, but should be easily translatable to VSCode launch.json files, Emacs dap-mode templates, etc.
+
+
+**NOTE: Some editors like Emacs using lsp-mode will require you to set a `noDebug` argument to `false`/`nil` to start a debug process, if not you will simply run the code.**
+
+
+**NOTE 2: A general tip for working with the debug adapter is building your codebase before launching the debugger. A lot of people experiencing "class not found"-style issues, and they are in most cases caused by not building before debugging. A simple Maven or Gradle compile should suffice.**
```

Comment:
Love this note. Took me a bit to get used to doing this.
One of the bits of magic :magic_wand: you will miss first when stepping away from Intelij